examples/data-objects/canvas/src/view.tsx

```diff
@@ -0,0 +1,147 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IColor, InkCanvas } from "@fluidframework/ink";
+
+import React, { useEffect, useRef, useState } from "react";
+
+import { Canvas } from "./canvas";
+// eslint-disable-next-line import/no-unassigned-import
+import "./style.less";
+
+const colorPickerColors: IColor[] = [
+    { r: 253, g: 0, b: 12, a: 1 },
+    { r: 134, g: 0, b: 56, a: 1 },
+    { r: 253, g: 187, b: 48, a: 1 },
+    { r: 255, g: 255, b: 81, a: 1 },
+    { r: 0, g: 45, b: 98, a: 1 },
+    { r: 255, g: 255, b: 255, a: 1 },
+    { r: 246, g: 83, b: 20, a: 1 },
+    { r: 0, g: 161, b: 241, a: 1 },
+    { r: 124, g: 187, b: 0, a: 1 },
+    { r: 8, g: 170, b: 51, a: 1 },
+    { r: 0, g: 0, b: 0, a: 1 },
+];
+
+interface IToolbarProps {
+    toggleColorPicker: () => void;
+    replayInk: () => void;
+    clearInk: () => void;
+}
+
+const Toolbar: React.FC<IToolbarProps> = (props: IToolbarProps) => {
```

Comment:
redundant typing
```suggestion
const Toolbar: React.FC<IToolbarProps> = (props) => {
```

Reply:
Ideally yes, but unfortunately eslint doesn't detect this correctly and errors:

https://github.com/yannickcr/eslint-plugin-react/issues/2353